src/AbpHelper.Core/Templates/Controller/Groups/Controller/src/{{ProjectInfo.FullName}}.HttpApi/{{InterfaceInfo.RelativeDirectory}}/{{Option.Name}}Controller.cs

```diff
@@ -13,29 +13,29 @@
 
 namespace {{ ProjectInfo.FullName }}.{{ InterfaceInfo.RelativeNamespace }}
 {
-    {{~ for attribute in ClassInfo.Attributes ~}}
-    {{~ if attribute | string.starts_with "[RemoteService"; defined_remote_service = attribute; end ~}}
-    {{~ end ~}}
-    {{~ if defined_remote_service ~}}
-    {{ defined_remote_service }}
-    {{~ else ~}}
-    [RemoteService(Name = "{{ ProjectInfo.Name }}{{ Option.Name }}")]
-    {{~ end ~}}
-    {{~ if ProjectInfo.TemplateType == 'Application' ~}}
-    [Route("/api/app/{{ Option.Name | abp.camel_case }}")]
-    {{~ else if ProjectInfo.TemplateType == 'Module' ~}}
-    [Route("/api/{{ ProjectInfo.Name | abp.camel_case }}/{{ Option.Name | abp.camel_case }}")]
-    {{~ end ~}}
-    public class {{ Option.Name }}Controller : {{ ProjectInfo.Name }}Controller, I{{ Option.Name }}AppService
-    {
-        private readonly I{{ Option.Name }}AppService _service;
+{{~ for attribute in ClassInfo.Attributes ~}}
+{{~ if attribute | string.starts_with "[RemoteService"; defined_remote_service = attribute; end ~}}
+{{~ end ~}}
+{{~ if defined_remote_service ~}}
+{{ defined_remote_service }}
+{{~ else ~}}
+[RemoteService(Name = "{{ ProjectInfo.Name }}{{ Option.Name }}")]
+{{~ end ~}}
+{{~ if ProjectInfo.TemplateType == 'Application' ~}}
+[Route("/api/app/{{ Option.Name | abp.kebab_case }}")]
+{{~ else if ProjectInfo.TemplateType == 'Module' ~}}
+[Route("/api/{{ ProjectInfo.Name | abp.kebab_case }}/{{ Option.Name | abp.kebab_case }}")]
```

Comment:
Undefined `kebab_case` method.

https://github.com/EasyAbp/AbpHelper.CLI/blob/ec35f00513d4b277b0dda1058e16ec6682734298/src/AbpHelper.Core/Generator/AbpFunctions.cs#L14-L23

![image](https://user-images.githubusercontent.com/13699353/169790878-6c901bab-c2dd-4c34-a99b-68c962a42a20.png)

Reply:
Thanks for testing.

I need your assist to decide which version of code I should do the copy & paste.

Previously I modified these codes base on [AbpHelper.GUI](https://github.com/EasyAbp/AbpHelper.CLI/blob/67bbde8d9f525820e63ca162a32263732ba16f05/src/AbpHelper.Core/Templates/Controller/Groups/Controller/src/%7B%7BProjectInfo.FullName%7D%7D.HttpApi/%7B%7BInterfaceInfo.RelativeDirectory%7D%7D/%7B%7BOption.Name%7D%7DController.cs) at line 25, 27. And I found out the there are differences on [AbpHelper.CLI](https://github.com/EasyAbp/AbpHelper.CLI/blob/develop/src/AbpHelper.Core/Templates/Controller/Groups/Controller/src/%7B%7BProjectInfo.FullName%7D%7D.HttpApi/%7B%7BInterfaceInfo.RelativeDirectory%7D%7D/%7B%7BOption.Name%7D%7DController.cs) at line 25, 27.

Reply:
This is a problem. Maybe we should merge other PRs first.

Reply:
@Hanazawa I am merging the PRs in https://github.com/EasyAbp/AbpHelper.CLI/labels/GUI-used-in-advance.

After I finish. We may need a new round of review to ensure that we have not modified the expired code in this PR.

Reply:
@Hanazawa I think we can merge the `EasyAbp:develop` branch into the current branch now.